staging/src/k8s.io/apimachinery/pkg/apis/meta/v1/validation/validation.go

```diff
@@ -104,7 +104,7 @@ func ValidateLabelSelectorRequirement(sr metav1.LabelSelectorRequirement, opts L
 func ValidateLabelName(labelName string, fldPath *field.Path) field.ErrorList {
 	allErrs := field.ErrorList{}
 	for _, msg := range validation.IsQualifiedName(labelName) {
-		allErrs = append(allErrs, field.Invalid(fldPath, labelName, msg).WithOrigin("labelKey"))
+		allErrs = append(allErrs, field.Invalid(fldPath, labelName, msg).WithOrigin("format=k8s-label-value"))
```

Comment:
These changes where needed, in order to let the tests pass for the core/validation tests.

Reply:
Why did you change it from "key" to "value"?  There's a different rule for each.  This is used to validatidate k8s label names, not values, and some test really do want to know the difference between "failed because name" and "failed because value"

Reply:
Sorry for the late reply. Had some busy work to-do at work.
I believe I failed on ready correctly here, as it was previously already "labelKey" - i don't know why I choosed k8s-label-value. I will fix this.